.github/workflows/strr-application-auto-approval-job-ci.yaml

```diff
@@ -0,0 +1,20 @@
+name: STRR APPLICATION AUTO APPROVAL JOB CI
+
+on:
+  pull_request:
+    paths:
+      - "jobs/auto-approval/**"
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+    working-directory: ./jobs/auto-approval
+
+jobs:
+  strr-pay-ci:
+    uses: bcgov/bcregistry-sre/.github/workflows/backend-ci.yaml@main
+    with:
+      app_name: "strr-application-auto-approval-job"
+      working_directory: "./jobs/auto-approval/**"
+      codecov_flag: "strrpaystrrapplicationautoapprovaljob"
```

Comment:
strrpaystrrapplicationautoapprovaljob not sure, is this the intended flag? name looks long, plus has strrpay prefix

Reply:
can i name it? i didn't have a name picked out preference would to name it auto-approval

Reply:
you can set codecov_flag to any name you like.
in the pipeline it is used as input for codecov/codecov-action@v4 plugin
https://github.com/bcgov/bcregistry-sre/blob/9a09e9ca57e2fd4fb30b3f73fe69093774f56cc0/.github/workflows/backend-ci.yaml#L150 in the docs for plugins it says that value is optional name to group codecov reports: https://github.com/codecov/codecov-action